test/stability/testDivide/testDivide.go

```diff
@@ -85,36 +80,59 @@ var (
 	fixMACAddrs1 func(*packet.Packet, flow.UserContext)
 	fixMACAddrs2 func(*packet.Packet, flow.UserContext)
 
-	rulesString       = "test-split.conf"
+	rulesString       = ""
 	filename          = &rulesString
 	passed      int32 = 1
+	l3Rules     *packet.L3Rules
+
+	gTestType uint
+	lessPercent uint
+	eps uint
 )
 
 func main() {
 	var testScenario uint
-	flag.UintVar(&testScenario, "testScenario", 0, "1 to use 1st part scenario, 2 snd, 0 to use one-machine test")
+	var testType uint
+	flag.UintVar(&testScenario, "testScenario", 0, "1 to use 1 as GENERATE part, 2 to use as RECEIVE part, 0 to use as ONE-MACHINE variant")
+	flag.UintVar(&testType, "testType", 0, "0 to use Separate test, 1 to use Split test, 2 to use Partition test")
 	flag.Uint64Var(&passedLimit, "passedLimit", passedLimit, "received/sent minimum ratio to pass test")
 	flag.Uint64Var(&speed, "speed", speed, "speed of generator, Pkts/s")
 	flag.UintVar(&outport1, "outport1", 0, "port for 1st sender")
 	flag.UintVar(&outport2, "outport2", 1, "port for 2nd sender")
 	flag.UintVar(&inport1, "inport1", 0, "port for 1st receiver")
 	flag.UintVar(&inport2, "inport2", 1, "port for 2nd receiver")
 	flag.Uint64Var(&totalPackets, "number", totalPackets, "total number of packets to receive by test")
-	flag.DurationVar(&T, "timeout", T, "test start timeout. Packets sent during timeout do not affect test result")
+	flag.DurationVar(&T, "timeout", T, "test start delay, needed to stabilize speed. Packets sent during timeout do not affect test result")
 	configFile := flag.String("config", "", "Specify json config file name (mandatory for VM)")
 	target := flag.String("target", "", "Target host name from config file (mandatory for VM)")
 	filename = flag.String("FILE", rulesString, "file with split rules in .conf format. If you change default port numbers, please, provide modified rules file too")
```

Comment:
btw do we need this flag? can use only rulesString var?

Reply:
We need it, rules for split and separate are differnet